js/component-courses.js

```diff
@@ -1,2 +1,40 @@
+
+// let courses = '<ul id="app-course">' +
+//         '<li v-for="course in list">{{ course }}</li>' +
+//         '</ul>';
+// let list;
+
+// /* Fetch the returned json object value. */
+// fetch('get_info.php').then(function (response) {
+//     list = JSON.parse(response);
+// });
+
```

Comment:
Don't leave commented out code in PRs

Reply:
Thanks, I've already removed them in the new commit!